pkg/slack/events/helpdesk/helpdesk-message.go

```diff
@@ -50,27 +45,37 @@ func Handler(client messagePoster, keywordsConfig KeywordsConfig) events.Partial
 			if event.ChannelType != "channel" {
 				return false, nil
 			}
-			if event.Channel != channelId {
-				log.Debugf("not in correct channel. wanted: %s, message was in: %s", channelId, event.Channel)
+			if event.Channel != forumChannelId {
+				log.Debugf("not in correct channel. wanted: %s, message was in: %s", forumChannelId, event.Channel)
 				return false, nil
 			}
-			if !strings.Contains(event.Text, helpdeskMention) {
+
+			var response []slack.Block
+			notifyToUseWorkflow := requireWorkflowsInForum && event.ThreadTimeStamp == "" && event.BotID == ""
```

Comment:
Is `event.ThreadTimeStamp == "" && event.BotID == ""` the indicator that workflow is not used?

Reply:
the `BotID` shows us that either a workflow was used or the post was from some bot (which currently isn't a thing that occurs in #forum-testplatform), and the lack of  `ThreadTimeStamp` shows us that it is a top-level message and not a reply.